nats.go

```diff
@@ -2692,7 +2692,10 @@ func (nc *Conn) processMsg(data []byte) {
 	}
 
 	// Copy them into string
-	subj := string(nc.ps.ma.subject)
+	subj := sub.Subject
+	if subj != string(nc.ps.ma.subject) {
```

Comment:
But for wildcard, are 2 copies made here? since we call string(nc.ps.ma.subject) twice? So yes, maybe evaluating if the subscription is wildcarded when created (so only once) and use that here would be better.

Reply:
@kozlovic no because of golang compiler optimisation